src/common/utility_unix.cpp

```diff
@@ -47,14 +47,22 @@ QString getUserAutostartDir_private()
 
 bool hasLaunchOnStartup_private(const QString &appName)
 {
-    QString desktopFileLocation = getUserAutostartDir_private() + appName + QLatin1String(".desktop");
+    QString desktopFileLocation = getUserAutostartDir_private()
+                                    + QLatin1String(APPLICATION_REV_DOMAIN)
```

Comment:
if you refactored this into, say, `APPLICATION_ID` then you'd save duplicated code and adhered more to the terminology in the standard, i.e. "appid".